app/presenters/monitoring-stations/view.presenter.js

```diff
@@ -51,19 +52,19 @@ function go(monitoringStation, auth) {
     pageTitle: _pageTitle(riverName, monitoringStationName),
     permissionToManageLinks: auth.credentials.scope.includes('manage_gauging_station_licence_links'),
     permissionToSendAlerts: auth.credentials.scope.includes('hof_notifications'),
-    restrictionHeading: restrictionHeading(licenceMonitoringStations),
-    restrictions: _restrictions(licenceMonitoringStations, FeatureFlagsConfig.enableLicenceMonitoringStationsView),
+    restrictionHeading: determineRestrictionHeading(licenceMonitoringStations),
+    restrictions: _restrictions(licenceMonitoringStations, FeatureFlagsConfig),
```

Comment:
Sorry, I meant this (along with my other suggested change)

```suggestion
    restrictions: _restrictions(licenceMonitoringStations),
```